### DISCUSSION TITLE
feature request: use exa for file preview

### DISCUSSION THREAD

**Discussion by basbebe** [PatrickF1/fzf.fish#117] [Archive] [answered]
use `exa --colors=always --grid --icons` instead of `ls` (or make it optional)

**Comment by PatrickF1** [comment, marked as answer]
Hi @basbebe , yes I will do this. This was also brought up a while ago. Linking the old discussion for full transparency https://github.com/PatrickF1/fzf.fish/discussions/101

**Reply by PatrickF1** [reply]
This is now possible! See #120
